src/flavors/eu/components/ecl-footers-eu/package.json

```diff
@@ -1,16 +1,13 @@
 {
   "name": "@ec-europa/ecl-footers-eu",
-  "version": "0.0.1",
-  "description": "ECL Footers",
-  "main": "ecl-footers.scss",
-  "style": "ecl-footers.scss",
   "author": "European Commission",
   "license": "EUPL-1.1",
+  "version": "0.0.1",
+  "description": "EU Footer",
+  "main": "ecl-footers-eu.scss",
+  "style": "ecl-footers-eu.scss",
```

Comment:
Not sure whether it worths keeping this `ecl-footers.scss` as the other one. Maybe we a discussion for naming of these files will be needed together with the overall naming topic around flavors/families.